packages/babel-parser/test/fixtures/experimental/_no-plugin/decimal/options.json

```diff
@@ -1,3 +1,4 @@
 {
-  "throws": "This experimental syntax requires enabling the parser plugin: \"decimal\". (1:2)"
-}
\ No newline at end of file
+  "BABEL_8_BREAKING": true,
+  "throws": "Identifier directly after number. (1:2)"
+}
```

Comment:
Since Babel 8 no longer supports parsing 0.1m and it is not expected to throw missing plugin error, we can just remove this case.